config/install/field.storage.node.field_pilots.yml

```diff
@@ -0,0 +1,18 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_pilots
+field_name: field_pilots
+entity_type: node
+type: entity_reference
+settings:
+  target_type: node
+module: core
+locked: false
+cardinality: 1
```

Comment:
This field should accept Unlimited values.
![vehicles_field_pilots_cardinality](https://github.com/notS0Solo/swapimigration/assets/1434979/28531ce1-d0d8-466c-ae0e-7aefd0b37248)

Reply:
Fixed